### PR TITLE
release v0.27.0

### PR DESCRIPTION
**Pull request by alexbrt**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 # Unreleased 
 - Alow users to filter get datasets by sources that they reference
```
Comment (annesylvie) [inline CHANGELOG.md:3]: You should add this to `0.27.0` as you are releasing it :)

**Review by annesylvie — COMMENTED** [review]
You also need to update the `README` with the latest images that will get built - see https://github.com/reinfer/cli/pull/277/files

File: CHANGELOG.md (hunk 1 above)
Comment (alexbrt) [reply CHANGELOG.md]: Oh that's right, thanks!

**Review by annesylvie — APPROVED** [review]
:rocket: Thanks! (Please squash your commits before merging!)

**Pull request merged**